beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -735,11 +639,14 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
             store.pruning_checkpoint_store_op(new_finalized_checkpoint),
```

Comment:
Lets delete the pruning_checkpoint while we're here.

https://github.com/sigp/lighthouse/blob/b5f27610ca675d649bbff755b970cea74ba144ad/beacon_node/store/src/hot_cold_store.rs#L2389-L2400

Reply:
Done in c8d82f08b